client/src/components/CBtn.style.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+const Button = styled.button`
+  width: 168px;
+  height: 32px;
```

Comment:
Styles look great! If we could convert the width and height to ```rem```s that's the only thing that I would change! 😄